scripts/generate-react-exports.js

```diff
@@ -24,43 +24,41 @@ const baklavaReactFileParts = [];
 
 for (const module of customElementsModules) {
   const { declarations, path } = module;
-  const componentDeclaration = declarations.find(declaration => declaration.customElement === true);
-  const { events, name: componentName, tagName: fileName, jsDoc } = componentDeclaration;
+  const component = declarations.find(declaration => declaration.customElement === true);
+  const { name: componentName, tagName: fileName } = component;
 
-  const eventNames =
-    events?.reduce((acc, curr) => {
-      acc[getReactEventName(curr.name)] = curr.name;
-      return acc;
-    }, {}) || {};
+  const eventNames = (component.events || []).map(event => {
+    const reactEvent = getReactEventName(event.name);
+      const reactEventName = `${componentName}${reactEvent.split("onBl")[1]}`;
+      return `${reactEvent}: '${event.name}' as EventName<${reactEventName}>`;
+    }).join(',\n');
 
-  const eventTypes =
-    events?.map(event => {
-      const eventName = getReactEventName(event.name);
-      const eventType = cleanGenericTypes(componentDeclaration.typeParameters, event.type.text);
-      const predefinedEventName = `${componentName}${eventName.split("onBl")[1]}`;
+  component.events?.forEach(event => {
+    const eventName = getReactEventName(event.name);
+    const eventType = cleanGenericTypes(component.typeParameters, event.type.text);
+    const predefinedEventName = `${componentName}${eventName.split("onBl")[1]}`;
 
-      eventStatements.push(`export declare type ${predefinedEventName} = ${eventType};`);
-      return `${eventName}: EventName<${predefinedEventName}>`;
-    }) || [];
+    eventStatements.push(`export declare type ${predefinedEventName} = ${eventType};`);
```

Comment:
So you just rename `CustomEvent<string>` as `BlTextareaInput` here, right?

Reply:
The output of this is
```tsx
export declare type BlTextareaInput = CustomEvent<string>;
```